com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TimeProfiler.java

```diff
@@ -203,9 +165,8 @@ public static String[] getHtmlContent() throws IOException {
         return new String[]{beforeToken, afterToken};
     }
 
-    public static void createReport(Boolean fromEditor) {
-        // avoid double creation of the report by checking `fromEditor` flag
```

Comment:
Since we now have a unified report creation place for editor and bob usage, there is no way to accidentally create the report twice.